docs/indexer/custom-processors/e2e-tutorial.md

```diff
@@ -73,107 +64,113 @@ For mac, if you're using brew, start it up with:
+Create your database with the name `coin_flip`, where our username is `user` and our password is `password`. If you are using `DBeaver`, you can create a new database by right clicking on the `Databases` tab and selecting `Create New Database`. 
 
-If your database is set up correctly, and you have the `psql` tool, you should be able to run the command `psql -d coin_flip`.
+If your database is set up correctly, and you have the `psql` tool, you should be able to run the command `psql coin_flip`.
 
-### Set up your local environment with poetry and grpc
+### Setup your local environment with poetry and grpc
 
 If you haven't yet, make sure to read the introductory [custom processor guide](https://github.com/aptos-labs/aptos-indexer-processors).
 
 You can also check out the python-specific broad overview of how to create an indexer processor [here](https://github.com/aptos-labs/aptos-indexer-processors/tree/main/python).
 
 ## Configure your indexer processor
 
-Now lets set up the configuration details for the actual indexer processor we're going to use.
+Now let's setup the configuration details for the actual indexer processor we're going to use.
 
-### Set up your config.yaml file
+### Setup your config.yaml file
 
 Copy the contents below and save it to a file called `config.yaml`. Save it in the `coin_flip` folder. Your file directory structure should look something like this:
 
 ```
-- indexer
+- aptos-indexer-processor
     - python
-        - aptos_ambassador_token
-        - aptos-tontine
-        - coin_flip
-            - move
-                - sources
-                    - coin_flip.move
-                    - package_manager.move
-                - Move.toml
-            - config.yaml     <-------- Edit this config.yaml file
-            - models.py
-            - processor.py
-            - README.md
-        - example_event_processor
-        - nft_marketplace_v2
-        - nft_orderbooks
-        __init__.py
-        main.py
-        README.md
+        - processors
+            - aptos_ambassador_token
+            - aptos-tontine
+            - coin_flip
+                - move
+                    - sources
+                        - coin_flip.move
+                        - package_manager.move
+                    - Move.toml
+                - config.yaml     <-------- Create and edit this config.yaml file
+                - models.py
+                - processor.py
+                - README.md
+            - example_event_processor
+            - nft_marketplace_v2
+            - nft_orderbooks
+            __init__.py
+            main.py
+            README.md
     - rust
     - scripts
     - typescript
 ```
 
-Once you have your config.yaml file open, you only need to change one field, if you just want to run the processor as is:
-
+Once you have your config.yaml file open, you only need to update a certain fields
```

Comment:
There's a hanging `a`

What are the certain fields?  Why do I want to update the certain fields?  I think it's a little unclear

```suggestion
Once you have your config.yaml file open, you only need to update certain fields
```